Lesson_12Hom/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <!-- <link rel="icon" href="#" type="images/x-icon"> -->
```

Comment:
why commented?

Reply:
Prepared thePrepared the code. until I made a small logo. code. until I made a small logo.